lib/PrintAsObjC/PrintAsObjC.cpp

```diff
@@ -944,10 +956,27 @@ class ObjCPrinter : private DeclVisitor<ObjCPrinter>,
              cast<ValueDecl>(D)->isInstanceMember()))
           continue;
         
-        if (isa<FuncDecl>(candidate) &&
-            (cast<FuncDecl>(candidate)->getParameters()->size() !=
-             cast<FuncDecl>(D)->getParameters()->size()))
-          continue;
+        if (isa<FuncDecl>(candidate)) {
+          auto cParams = cast<FuncDecl>(candidate)->getParameters();
+          auto dParams = cast<FuncDecl>(D)->getParameters();
```

Comment:
I hate to scope-creep now, but should this handle any AbstractFunctionDecl, which would also cover initializers? (And in theory getters and setters but people don't usually put renames on those individually.)

Reply:
@jrose-apple you mean should we change from FuncDecl to be AbstractFuncDecl? Did I understand your point correctly?

Reply:
Yes, that's what I was suggesting!

Reply:
Got it. Will try it on tomorrow and also look into the log about build failure too